cypress/support/index.d.ts

```diff
@@ -2,7 +2,10 @@
 
 declare namespace Cypress {
   interface Chainable<Subject> {
-    getByDataCy(selector: string): Chainable<any>
-    register(email: string, username: string, password: string): Chainable<any>
+    getByDataQa(selector: string): Chainable<any>;
```

Comment:
It's recommended to include comments above each method to briefly explain what each method does. This will improve code readability and maintenance.